.github/workflows/go.yml

```diff
@@ -2,9 +2,13 @@ name: Go
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - main
+      - dev
   pull_request:
-    branches: [ main ]
+    branches:
```

Comment:
Do we want to run this on every pull request, just as a matter of course?

It doesn't block anything just by running, but I think it'd ~always be useful. (And, not that we have any other target branches, but.)

Reply:
Oh yeah, now that you mention it, I *would* like tests to run on every PR. I'll patch to run on all base branches before I merge this.

Reply:
760c32ba9c6f03d45b369151bf4f48b6b2881c91